store/modules/getters/user.js

```diff
@@ -12,7 +12,11 @@ export const getUserIsRegistered = state => !!state.user.user;
 
 export const getUserIsInBonusCoolDown = state => state.user.bonusCooldown;
 
-export const getUserHasWallet = state => !!(state.user.wallet || state.user.cosmosWallet);
+export const getUserHasWallet = state => !!(
+  state.user.wallet || state.user.cosmosWallet || state.user.likeWallet
```

Comment:
Can we remove `state.user.wallet`? This function would be more useful without eth wallet
We need to make sure metamask login flow does not use this function though

Reply:
Can we just remove all ETH-related UI except for user login? Otherwise, that will make a lot of change

Reply:
hmm keep this change and leave for revamp?